pwnlib/tubes/process.py

```diff
@@ -740,10 +741,16 @@ def close(self):
         # First check if we are already dead
         self.poll()
 
-        #close file descriptors
+        # close file descriptors
         for fd in [self.proc.stdin, self.proc.stdout, self.proc.stderr]:
             if fd is not None:
-                fd.close()
+                try:
+                    fd.close()
+                except IOError as e:
+                    if e.errno == errno.EPIPE:
+                        pass
+                    else:
+                        print(e)
```

Comment:
A concise flow would be better.
```suggestion
                    if e.errno != errno.EPIPE:
                        raise
```